### ISSUE TITLE
feat: add plugin demo gif

### ISSUE THREAD

**Issue by vivifalls** [chadcat7/prism#1]
Adding to the README a short gif that demonstrates what the plugin does and the built-in colorschemes in it would be a nice small addition.

**Comment by namishh** [comment]
you are right, i was thinking of doing something like this. maybe in a day or two.
